xml/OrganisationRegistrationAgency.xml

```diff
@@ -6,17 +6,15 @@
         <description>
             <narrative>
                
-        Organization identifier lists (formerly Organization Registration Agency).
         The values from this codelist are used to identify the particular list that
-        an organization identifier was drawn from. The codelist provides a list of 
+        an organisation identifier was drawn from. The codelist provides a list of
         known identifier lists, including national company registers, NGO lists and
         international and multilateral lists - along with links to sources where 
-        specific organization numbers and identifiers can be looked up.
+        specific organisation numbers and identifiers can be looked up.
         
         As of 17th of July 2017 the Organisation Registration Agency codelist (now organisation
-        identifier list) is maintained centrally by org-id.guide. org-id.guide provides
-        a register of organisation identifier lists that allows open data
-        publishers to locate organisation identifiers. IATI replicates the
+        identifier list) is maintained centrally by [org-id.guide](http://org-id.guide). Open data
```

Comment:
Codelist narratives are plaintext rather than markdown. As such...

`[org-id.guide](http://org-id.guide)` -> `http://org-id.guide`

Reply:
fixed, wrapped and squashed